consensus/types/proto/block.proto

```diff
@@ -14,7 +14,7 @@ message BlockHeader {
   uint32 numTxs = 5;
   int64 totalTxs = 6; // Total = total in the entire chain Num = total in block (Tendermint legacy)
   string lastBlockHash = 7;
-  bytes proposerAddress = 8;
+  bytes proposerAddress = 8; // TODO: Change this to a string
```

Comment:
Not actionable, only a "stupid question".

Just for my understanding: we are going to prefer strings to byte arrays for anything that's addresses and hashes, correct?

Is it for simplicity or there are other reasons?

Reply:
> Is it for simplicity or there are other reasons?

1. Simplicity - they're small, not big data blobs
2. Readability - Things like `address` or `stateHash` are "human readable" in the sense that we use & reference it (e.g. printing)
3. Performance (micro optimization) - lots of unnecessary encoding & decoding throughout the codebase